geomesa-geojson/geomesa-geojson-gs-plugin/pom.xml

```diff
@@ -26,6 +26,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-accumulo-datastore_${scala.binary.version}</artifactId>
```

Comment:
How was this working previously!?

Reply:
previously the geojson module had a dependency on geomesa-accumulo, but that wasn't really necessary because it can work with any data store. So the gs-plugin will still be bundled with accumulo, but the api module is not.